README.md

```diff
@@ -5,6 +5,7 @@
 [![Apache 2.0 license](https://img.shields.io/github/license/sandialabs/gulliver?logo=apache)](https://github.com/sandialabs/Gulliver/blob/master/LICENSE)
 [![.NetStandard 1.3](https://img.shields.io/badge/targets-.NETStandard%201.3-5C2D91?logo=.net)](https://docs.microsoft.com/en-us/dotnet/standard/net-standard)
 [![Join the chat at https://gitter.im/sandialabs/Gulliver](https://badges.gitter.im/sandialabs/Gulliver.svg)](https://gitter.im/sandialabs/Gulliver?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+[![Build Status](https://dev.azure.com/sandianationallabs/Gulliver/_apis/build/status/sandialabs.Gulliver?branchName=master)](https://dev.azure.com/sandianationallabs/Gulliver/_build/latest?definitionId=1&branchName=master)
 
```

Comment:
how about code coverage
 ![Azure DevOps coverage](https://img.shields.io/azure-devops/coverage/sandianationallabs/Gulliver/1)

```markdown
![Azure DevOps coverage](https://img.shields.io/azure-devops/coverage/sandianationallabs/Gulliver/1)
```
and tests
![Azure DevOps tests (compact)](https://img.shields.io/azure-devops/tests/sandianationallabs/Gulliver/1?compact_message)

```markdown
![Azure DevOps tests (compact)](https://img.shields.io/azure-devops/tests/sandianationallabs/Gulliver/1?compact_message)
```
and

Reply:
yes! Let me add those and turn on the code coverage results, which are missing.

Reply:
@nicksterx, how is versioning handled for releases? Tagging?
Does it auto release on nugget for us?

Reply:
> yes! Let me add those and turn on the code coverage results, which are missing.

Cool, will review in a bit. I need to climb onto the roof and decommission the swamp cooler. If I don't approve within an hour call an ambulance / mortition.

Reply:
Haha I will!

Reply:
Now it's ready!